### DISCUSSION TITLE
Add Detection Filters to Birds of Week Page

### DISCUSSION THREAD

**Discussion by Phaeton** [mmcc-xx/BirdCAGE#14] [Ideas]
It would be nice to see what Detection Filters are applied to the birds listed on the "Birds of the Week" page as an additional column in the table. Yes, I know they are listed lower on the Detection Filters page, but it would be nice to be able to review at a glance, on the page where the birds are sorted by observation likelihood.

**Comment by mmcc-xx** [comment]
You mean like what category they are in? Ignore/Log/Record/Alert? That would be pretty nice. Being able to drag the threshold values around up and down the list would be slick, and being able to directly edit the overrides from there... Hmmm

**Reply by Phaeton** [reply]
> You mean like what category they are in? Ignore/Log/Record/Alert? That would be pretty nice.

Exactly what I was thinking. Your additional thoughts sound pretty slick as well.





> Side note - I'm not really sure what the sf_thresh value has on the actual analysis. I know I've gotten hits on birds that fall below the cutoff - like I've had a barred owl show up when its occurrence value is .027 and my cutoff was .03.

Technically that rounds to .03.... :grinning:

**Comment by mmcc-xx** [comment]
Side note - I'm not really sure what the sf_thresh value has on the actual analysis. I know I've gotten hits on birds that fall below the cutoff - like I've had a barred owl show up when its occurrence value is .027 and my cutoff was .03.

**Comment by Phaeton** [comment]
Had a couple thoughts over last few days that are marginally related to this thread. Fully acknowledge that I might be a edge case here, but was thinking the following:
- On the default page, it would be nice to always see spectrograms/recordings for the last 5 detections. Because I am currently throwing away log only recordings, I understand there is nothing for the spectrogram to be generated off of. Would it be hard to write the last 5 to a `/tmp` folder, reference these instead, and rotate through as needed? I acknowledge it goes against the thought of log only. I could set everything back to record or default, but I would only be doing so to generate the last 5 spectrograms/recordings which seems a bit excessive if I am trying to minimize writes to disk. The main reason I want to do this is to have a more consistent visual to the page. Currently some records have recordings/spectrograms, others do not.
- I like the option to keep best recording. Curious if we could add a check for log only to see if the recording is better than the best and if so, keep that recording. Again, not exactly log only, but I think having the best recordings, and eventually a page to view would be a nice addition.
